docs/SYNC_MODE.md

```diff
@@ -32,11 +32,68 @@ Taps can support two replication methods, and should decide which to use by chec
 ## Stream/Field Selection
 Taps should allow users to choose which streams and fields to replicate. The following metadata should be checked to decide whether a stream/field should be replicated:
 
-| Metadata Keyword | Description  | 
+| Metadata Keyword | Description  |
 | ----------------- | ------- |
 | `inclusion` | Only applies to fields.  If this is set to `automatic`, the field should be replicated.  If this is set to `unsupported`, the field should not be replicated.  Can be written by a tap during discovery |
 | `selected` | If this is set to `True`, the stream (empty breadcrumb), or field should be replicated.  If `False`, the stream or field should be omitted.  This metadata is written by services outside the tap. |
+| `available` | The field may or may not be `selected`. |
 
+Streams shouldn’t be selected by default. We want them to have an `inclusion: available` property. Each field needs to have its own metadata object that labels the field as `inclusion: automatic` or `inclusion: available`.
```

Comment:
Avoid using `we` like this. Try to write in an "Active" voice when providing documentation instructions like this.  See https://blogs.wp.missouristate.edu/writinglab/writing-process/active-and-passive-voice/ and https://softwareengineering.stackexchange.com/questions/190746/i-we-or-neither-in-code-documentation

Reply:
This is done more times below, can you take a stab at removing the we's in other places as well.

Reply:
I'd avoid language like "we" in these things. It's less of an opinion of the team and more of a requirement of the standards. So, I'd remove the "We want them to have ..." sentence.